src/main/java/com/minecolonies/coremod/entity/ai/citizen/smelter/EntityAIWorkSmelter.java

```diff
@@ -399,15 +406,76 @@ private int getRequiredProgressForMakingRawMaterial()
         return PROGRESS_MULTIPLIER / Math.min((int) (getPrimarySkillLevel()/2.0) + 1, MAX_LEVEL) * HITTING_TIME;
     }
 
-    private ItemStack extractEnchantFromItem(final ItemStack item)
+    private static ItemStack extractEnchantFromItem(final ItemStack item)
     {
         final Map<Enchantment, Integer> enchants = EnchantmentHelper.getEnchantments(item);
         final ItemStack books = new ItemStack(Items.ENCHANTED_BOOK);
         for (final Map.Entry<Enchantment, Integer> entry : enchants.entrySet())
         {
             EnchantedBookItem.addEnchantment(books, new EnchantmentData(entry.getKey(), entry.getValue()));
         }
-        worker.decreaseSaturationForAction();
         return books;
     }
+
+    @NotNull
+    public static List<IGenericRecipe> getAdditionalRecipesForDisplayPurposesOnly()
```

Comment:
I'm really unsure about all of this, basically, this is the #1 feature that I'm unsure about and wondering about scrapping/modifying it. That's why I'm unsure if I want people to actually see this so obvious in JEI atm.

Reply:
If the feature itself is removed, the JEI display can be removed at the same time (that's an advantage of having them both in the same place rather than in entirely different files).

While it exists, though, it ought to be documented in some way, otherwise nobody would know that it exists.

The other option might be to remove the JEI/crafting module entirely -- which is not unreasonable since the smelter doesn't support crafterrecipes or teachable recipes at all, but it would make the mod less self-documenting.  (I was considering later adding a JEI page for the animal huts too, showing the animal drops as outputs plus some special cases like cow milking.  This would use the same mechanism, although is a bit more complicated due to loot tables.)

Reply:
there is some stuff which is intentionally not shown/displayed as readily available public information such as wiki/jei is

Reply:
Sure, and that makes sense for some things, like specific formulas for raids and XP etc.  It makes less sense to hide crafting mechanics, though.